PowerEditor/src/ScitillaComponent/Buffer.cpp

```diff
@@ -1088,8 +1088,15 @@ bool FileManager::saveBuffer(BufferID id, const TCHAR * filename, bool isCopy, g
 		// Note that fwrite() doesn't return the number of bytes written, but rather the number of ITEMS.
 		if(items_written != 1)
 		{
-			if(error_msg != NULL)
-				*error_msg = TEXT("Failed to save file.\nNot enough space on disk to save file?");
+			if (error_msg != NULL)
+			{
+				NativeLangSpeaker *_nativeLangSpeaker = (NppParameters::getInstance())->getNativeLangSpeaker();

```

Comment:
The underscore prefix is used to indicate a class member, `_nativeLangSpeaker` is method/block local.

Reply:
So us suggest to rename it to `nativeLangSpeaker` ?

Reply:
`pNativeSpeaker` as is already used in e.g. `Notepad_plus.cpp`.

Reply:
OK, thank you @MAPJe71